angular/src/lib/alert/tests/__snapshots__/alert.service.spec.ts.snap

```diff
@@ -71,16 +71,11 @@ exports[`AlertService should render a cui-alert-container html element in bottom
                   style="opacity: 1;"
                 >
                   <cui-icon
+                    class="cui-icon icon icon-cancel_16"
```

Comment:
@pauljeter I removed the <i> wrapper of the IconComponent. The Icon in Avatar is not center aligned because of the two-level structure. Icon React component does not have a wrapper.